Application/Common/Models/AddToShortlistModel.cs

```diff
@@ -0,0 +1,6 @@
+namespace Application.Common.Models;
+
+public class AddToShortlistModel : ShortlistModelCommon
```

Comment:
Are these models only used for the tests? If so it might be worth moving them to the Tests namespace?

Reply:
The `AddToShortlistModel` and `RemoveFromShortlistModel` models are used to capture the data from the client side. For example, for the following `Task<IActionResult> OnPostAddToShortlist([FromBody] AddToShortlistModel shortlistModel)`, when the user adds a TP to the shortlist, we will be receiving post body data with the `AddToShortlistModel` model, and we then process that on the server side.